features/IBF-portal-user/dashboard-page/View_user_header_section.feature

```diff
@@ -0,0 +1,60 @@
+@ibf-portal-user
+Feature: View and use header section
+Background:
+    Given any logged in user
+
+Scenario: View header of dashboard page
+    When the user looged-in into the IBF-portal
+    Then the user redirected to IBF-dashboard
+    And it shows "Logged in as" with login country's name 
+    And it contains login country name (by using which country user looged in as)
+    And it contains menu-icon, logo and "Logout"  and "Export View" buttons.
+    And it contains that country's logo on the right side of the dashboard
+    And it contains Logout and Export-view buttons 
+
+
+Scenario: View logout page
+    Given any logged in user
```

Comment:
The point of given is that you don't need to repeat if every time, if you have already put it in 'background' on top.